bot/cogs/pavlov.py

```diff
@@ -47,7 +48,7 @@ def __init__(self, bot):
     async def on_ready(self):
         logging.info(f"{type(self).__name__} Cog ready.")
 
-    async def get_map_alias(self, map_label: str) -> [str, str]:
+    async def get_map_alias(self, map_label: str):
```

Comment:
Any reason why the return hint was removed? 😅

Reply:
```
List expression not allowed in type annotation
  Use List[T] to indicate a list type or Union[T1, T2] to indicate a union typePylance
  ``` VSCode kept throwing this error.

Reply:
I threw it back in and it seems to be running fine.